ui/hooks/metamask-notifications/useSwitchNotifications.test.tsx

```diff
@@ -68,8 +68,8 @@ describe('useSwitchNotifications', () => {
       ),
     });
 
-    act(() => {
-      result.current.onChange(true);
+    await act(async () => {
+      await result.current.onChange(true);
```

Comment:
Tests were fine, but we had `act` warnings. This ensures we wait for act blocks to finish.